source/_integrations/adax.markdown

```diff
@@ -14,13 +14,23 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The **Adax** {% term integration %} integrates Adax heater into Home Assistant.
+The **Adax** {% term integration %} integrates Adax heaters into Home Assistant and can be configured to use a local or cloud interface.
 
-You can configure the heaters to use a local or cloud interface.
+{% include integrations/config_flow.md %}
```

Comment:
This ordering makes no sense to me. The original one does. One need to configure their devices first, before setting up the integration in Home Assistant.

This has now been made backwards, which will lead to frustration and misunderstandings.

Reply:
Good point, moved.